### PR TITLE
gpt-2-simple: init at 0.8.1

### PR DESCRIPTION
**Pull request by ckiee**
###### Description of changes

For https://github.com/Xe/x/blob/master/mastodon/robocadey/gpt2/main.py

cc @Xe (::

-----

https://github.com/minimaxir/gpt-2-simple
https://pypi.org/project/gpt-2-simple

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ckiee** [comment]
Result of `nixpkgs-review pr 170713` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.gpt-2-simple</li>
    <li>python39Packages.gpt-2-simple</li>
  </ul>
</details>

**Comment by Xe** [comment]
+1 I have tested this in one of my own projects and this works perfectly.

File: pkgs/development/python-modules/gpt-2-simple/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pytestCheckHook, regex, requests
```
Comment (gador) [inline pkgs/development/python-modules/gpt-2-simple/default.nix:1]: pytestcheckhook isnt used

**Review by gador — CHANGES_REQUESTED** [review]
This is a semi-automatic executed nixpkgs-review with nixpkgs-review-checks extension. It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of nixpkgs-review pr 170713 run on x86_64-linux 1

<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python310Packages.gpt-2-simple</li>
<li>python39Packages.gpt-2-simple</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
python310Packages.gpt-2-simple:
<details>

got build log for '/nix/store/4lfh5rik5v8zq98xarfmyabg28hpz2nv-python3.10-gpt-2-simple-0.8.1' from 'daemon'
When evaluating attribute ‘python310Packages.gpt-2-simple’:
warning: no-python-tests
Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/gpt-2-simple/default.nix:20:

────────────────────
   |
20 |     description =
   | ^
────────────────────

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md
warning: unused-argument
Unused argument: pytestCheckHook.
Near pkgs/development/python-modules/gpt-2-simple/default.nix:1:45:

────────────────────
  |
1 | { lib, buildPythonPackage, fetchFromGitHub, pytestCheckHook, regex, requests
  |                                             ^
────────────────────

</details>
</details>

**Review by gador — APPROVED** [review]

**Pull request merged**